.backportrc.json

```
@@ -1,6 +1,6 @@
 {
   "upstream": "elastic/beats",
-  "branches": [{ "name": "7.9"}, { "name": "7.8"}, { "name": "7.7"}, { "name": "7.x"}],
+  "branches": [ { "name": "7.10"}, { "name": "7.9"}, { "name": "7.8"}, { "name": "7.7"}, { "name": "7.x"}],
```

Can we simplify this? And put 7.x first, with it automatically checked?

```suggestion
  "branches": [ { "name": "7.x", "checked": true }, "7.10", "7.9", "7.8", "7.7"],
```